docs/migration-guide/migration-guide-master-page.md

```diff
@@ -0,0 +1,48 @@
+# Migrating a SpatialOS project to the FPL
+This page presents a step-by-step guide on how to migrate a SpatialOS project from the [current project layout (CPL)](https://docs.improbable.io/reference/latest/shared/reference/project-structure) to the flexible project layout (FPL).
+
+## Overview
+To migrate a SpatialOS project from the CPL to the FPL, you need to:
+1. [Migrate your workers](#1-migrate-your-workers)
+2. [Migrate your launch configuration file](#2-migrate-your-launch-configuration-files)
+3. [Migrate your project definition file](#3-migrate-your-project-definition-file)
+4. [Generate a schema descriptor](#4-generate-a-schema-descriptor)
+5. [Verify the integrity of your project](#5-verify-the-integrity-of-your-project)
+6. [Customise your project folder structure](#6-customise-your-project-folder-structure)
+
+**Note:** Migrating a SpatialOS project from CPL to FPL is an all or nothing process. A project must either fully follow the CPL format or the FPL format and cannot have partially migrated components.
+
+**Tip:** We recommend that you back up a copy of your project before you start migrating so that you can recover information from files that you delete as part of the migration process if needed.
+
+## 1. Migrate your workers
+1. Migrate the build process of your workers by following the steps in [Building a worker executable](../build-process/worker-build-process.md).
+2. Migrate your workers’ worker configuration files by following the steps in [Converting the worker configuration file](configs-conversion-guide.md#converting-the-worker-configuration-file).
+3. Delete all previous CPL format configuration files related to the configuration and build process of your workers. Those configuration files include:
+    - Your CPL format [worker configuration files](https://docs.improbable.io/reference/latest/shared/worker-configuration/worker-configuration)
```

Comment:
Just realised that the "worker configuration file" is a single file, so it doesn't make sense to use the plural here, or to have the second bullet point (because the build config is part of the worker config file, not a separate file).

Reply:
It's one file per worker but potentially multiple files per project (if you have multiple workers).